src/test/java/com/qaprosoft/carina/demo/phpTravels/LoginPageTest.java

```diff
@@ -25,68 +26,64 @@ public void testForgotAccButton() {
 
     @Test
     public void testLoginAdminAcc() {
-        loginIn("admin@phptravels.com", "demoadmin");
+        login();
         DashboardPage dashboardPage = new DashboardPage(getDriver());
         Assert.assertTrue(dashboardPage.isPageOpened());
     }
 
     @Test
     public void testChangeName() {
-        loginIn("admin@phptravels.com", "demoadmin");
+        login();
         DashboardPage dashboardPage = new DashboardPage(getDriver());
-        LeftMenuBar leftMenu =dashboardPage.getLeftMenuBar();
+        SettingsPage settingsPage = dashboardPage.getLeftMenuBar().openSettingsSubmenu();
 
-        SettingsPage settingsPage = leftMenu.openSettingsSubmenu();
         settingsPage.typeNameText("JS Travels | Travel Technology Partner");
         settingsPage.clickSaveChangeButton();
-        Assert.assertTrue(settingsPage.getChangesSavedText());
+
+        Assert.assertTrue(settingsPage.getChangesSaved());
     }
 
     @Test
     public void testUnpaidBookingsMenu() {
-        loginIn("admin@phptravels.com", "demoadmin");
+        login();
         DashboardPage dashboardPage = new DashboardPage(getDriver());
-        NavigationBar navigationBar = dashboardPage.getNavigationBar();
-
-        BookingsPage bookingsPage = navigationBar.openBookingsPage();
-        BookingsMenu bookingsMenu = bookingsPage.getBookingsButtons();
+        BookingsMenu bookingsMenu = dashboardPage.getNavigationBar().openBookingsPage().getBookingsMenu();
+        BookingsPage bookingsPage = new BookingsPage(getDriver());
         bookingsMenu.clickUnpaidBookings();
```

Comment:
It can be done in 1 line, your clickUnpaidBookings method returns BookingsPage
BookingsPage bookingsPage = bookingsMenu.clickUnpaidBookings();